test/test.js

```diff
@@ -648,119 +676,34 @@ tape("Winston supports json logging.", function (t) {
     ]
   });
 
-  mockTest(function (data) {
-    t.pass("winston logs in json format");
-    const expect = {
-      message: "msg",
-      foo: "bar",
-      level: "warn"
-    };
-    t.equal(data, x + " " + JSON.stringify(expect) + '\n', 'json as expected');
-  });
-
-  logger.warn("msg", { foo: "bar" });
-});
-
-tape('Winston@1.1.2 integration is provided.', function (t) {
-  t.plan(4);
-  t.timeoutAfter(2000);
-
-  t.true(winston1.transports.Logentries,
-      'provisioned constructor automatically');
-
-  t.doesNotThrow(function () {
-    winston1.add(winston1.transports.Logentries, { token: x, region: 'eu' });
-  }, 'transport can be added');
-
-  winston1.remove(winston1.transports.Console);
-
-  mockTest(function (data) {
-    t.pass('winston log transmits');
-    t.equal(data, x + ' warn mysterious radiation\n', 'msg as expected');
-  });
-
-  winston1.warn('mysterious radiation');
-});
-
-tape("Winston@1.1.2 supports json logging.", function (t) {
-  t.plan(2);
-  t.timeoutAfter(2000);
-
-  const logger = new (winston1.Logger)({
-    transports: [
-      new (winston1.transports.Logentries)({ token: x, json: true, region: 'eu' })
-    ]
-  });
 
   mockTest(function (data) {
     t.pass("winston logs in json format");
     const expect = {
-      message: "msg",
       foo: "bar",
-      level: "warn"
+      level: "warn",
+      message: "msg",
     };
     t.equal(data, x + " " + JSON.stringify(expect) + '\n', 'json as expected');
   });
 
   logger.warn("msg", { foo: "bar" });
 });
 
-tape('Winston@2.1.1 integration is provided.', function (t) {
-  t.plan(4);
-  t.timeoutAfter(2000);
-
-  t.true(winston2.transports.Logentries,
-      'provisioned constructor automatically');
-
-  t.doesNotThrow(function () {
-    winston2.add(winston2.transports.Logentries, { token: x, region: 'eu' });
-  }, 'transport can be added');
-
-  winston2.remove(winston2.transports.Console);
-
-  mockTest(function (data) {
-    t.pass('winston log transmits');
-    t.equal(data, x + ' warn mysterious radiation\n', 'msg as expected');
-  });
-
-  winston2.warn('mysterious radiation');
-});
-
-tape("Winston@2.1.1 supports json logging.", function (t) {
-  t.plan(2);
-  t.timeoutAfter(2000);
-
-  const logger = new (winston2.Logger)({
-    transports: [
-      new (winston2.transports.Logentries)({ token: x, json: true, region: 'eu' })
-    ]
-  });
-
-  mockTest(function (data) {
-    t.pass("winston logs in json format");
-    const expect = {
-      message: "msg",
-      foo: "bar",
-      level: "warn"
-    };
-    t.equal(data, x + " " + JSON.stringify(expect) + '\n', 'json as expected');
-  });
 
-  logger.warn("msg", { foo: "bar" });
-});
 
 // BUNYAN STREAM
 
 tape('Bunyan integration is provided.', function (t) {
   t.plan(9);
 
-  const streamDef = Logger.bunyanStream({ token: x, minLevel: 3, region: 'eu' });
+  const streamDef = buildBunyanStream({ token: x, minLevel: 3, region: 'eu' });
```

Comment:
Would be nice if we could keep `Logger.bunyamStream` too as to not unnecessarily break people's integrations

Reply:
I guess it would be nice from a code perspective but conceptually I wouldn't agree.
Before, everything was dumped in the same file so we could do this, but I split out all the components and I'd argue we should keep un-related items in different files since it's easier to manage. Everything related to BunyanStream is in its own file.

BunyanStream is built on-top of Logger, so for me it's a lot cleaner to have small units build on-top of eachother.
Also, major release so it's okay to add breaking changes and it's not like we're "unnecessarily" breaking it, we have a reason

Reply:
Another reason this should be split out completely is because I ran into so many circular dependency issues.

Reply:
I think this is acceptable now that we have proper release versioning.

Reply:
But we should still export every public method/"class" from the entrypoint (`index.js`?) so that customer integrations don't break. Customers should also never need to do something like `require('r7insight_node/some_file')`.

Reply:
Yup, you're right, just forgot to export it in the entrypoint